packages/larva-patterns/objects/o-checkbox-input-list/o-checkbox-input-list.twig

```diff
@@ -1,4 +1,5 @@
-<div class="{{ o_checkbox_input_list_classes }}">
+<section class="o-checkbox-input-list {{ o_checkbox_input_list_classes }}">
```

Comment:
Do we need `o-checkbox-input-list` class for objects as well?

Reply:
Yes - we need to be able to tell what pattern it is when inspecting the markup - otherwise it will just be utilities. Thanks!